docs/reference/release-notes/7.17.asciidoc

```diff
@@ -0,0 +1,136 @@
+
+License::
+* Add DEBUG log when DLS/FLS usage is detected {es-pull}82182[#82182] (issue: {es-issue}79152[#79152])
+
+Machine Learning::
+* Move datafeed stats action off of master node {es-pull}82271[#82271]
+
+Watcher::
+* Preventing watcher from starting up if its templates are missing {es-pull}82395[#82395]
+
+
+
+[[bug-7.17.0]]
+[float]
+=== Bug fixes
+
+Aggregations::
+* Fix bucket keys format for range aggregations on float field {es-pull}81801[#81801] (issue: {es-issue}81749[#81749])
+* Fix cardinality agg in async search {es-pull}82108[#82108]
+* Fix: missing fields in range aggregation response for date fields {es-pull}82732[#82732] (issue: {es-issue}82688[#82688])
+
+Authorization::
+* Fix FLS for frozen tier {es-pull}82521[#82521] (issues: {es-issue}78988[#78988], {es-issue}82044[#82044])
+
+CRUD::
+* Fix potential listener leak in `TransportBulkAction` {es-pull}81894[#81894]
+
+Data streams::
+* Change `DataStream` class to have a single public constructor {es-pull}82464[#82464]
+
+Geo::
+* Handle degenerated rectangles in vector tiles {es-pull}82404[#82404] (issue: {es-issue}81891[#81891])
+* `GeoPolygonDecomposer` might fail due to numerical errors when calculating intersection with the dateline {es-pull}82953[#82953] (issue: {es-issue}82840[#82840])
+
+ILM+SLM::
+* Data tiers: migrate the coldest node attribute {es-pull}81940[#81940] (issue: {es-issue}81633[#81633])
+* ILM step retry safe refresh of the cached phase {es-pull}82613[#82613] (issue: {es-issue}81921[#81921])
+* ILM: fix allocate action to allow only `total_shards_per_node` {es-pull}81944[#81944] (issue: {es-issue}81943[#81943])
+
+Indices APIs::
+* Make the rollover API respect the request's `master_timeout` {es-pull}82326[#82326] (issue: {es-issue}81762[#81762])
+
+Infra/Core::
+* Allow clearing blocks on managed system indices {es-pull}82507[#82507] (issue: {es-issue}80814[#80814])
+
+Infra/Logging::
+* Add `doPrivileged` section in deprecation logger {es-pull}81819[#81819] (issue: {es-issue}81708[#81708])
+
+Ingest::
+* Filter enrich policy index deletes to just the policy's associated indices {es-pull}82568[#82568]
+* Fix enrich cache corruption bug {es-pull}82441[#82441] (issue: {es-issue}82340[#82340])
+
+Java Low Level REST Client::
+* Fix version resolution and encoding in LLRC {es-pull}81989[#81989]
+
+Machine Learning::
+* Fix annotations index maintenance after reindexing {es-pull}82304[#82304] (issue: {es-issue}82250[#82250])
+* Improve cleanup for model snapshot upgrades {es-pull}81831[#81831] (issue: {es-issue}81578[#81578])
+* Make delete intervening results more selective {es-pull}82437[#82437]
+* Skip time to next interval with data for datafeeds with aggs {es-pull}82488[#82488] (issue: {es-issue}82406[#82406])
+* Update running process when global calendar changes {es-pull}83044[#83044]
+
+Monitoring::
+* Always attempt upgrade monitoring templates {es-pull}82713[#82713] (issue: {es-issue}82453[#82453])
+
+Other::
```

Comment:
```suggestion
Settings::
```

Reply:
After renaming this section to **Settings**, I'd move it after line 121 to keep the alphabetic ordering.